cdap-ui/server/config/ui-settings.json

```diff
@@ -1,4 +1,4 @@
 {
-  "standalone.welcome.message": "true",
+  "standalone.welcome.message": "false",
```

Comment:
This should not be changed either. The default is to show welcome screen for now.